sunpy/visualization/animator/image.py

```diff
@@ -79,15 +79,16 @@ def plot_start_image(self, ax):
         # Create extent arg
         extent = []
         # reverse because numpy is in y-x and extent is x-y
+        if max([len(self.axis_ranges[i]) for i in self.image_axes[::-1]]) > 2:
```

Comment:
```suggestion
        if max(map(len, self.image_axes[::-1])) > 2:
```
But why iterate backwards? Shouldn't this be the same than:
```python
        if max(map(len, self.image_axes)) > 2:
```

Reply:
You're right.  It should be.  It may be legacy.  But I'll have to check...